api/resources_portal/test/test_data.json

```diff
@@ -0,0 +1,701 @@
+      "grant_id": "1",
+      "material_id": "1"
+    },
+    "1": {
+      "id": "2",
+      "grant_id": "1",
+      "material_id": "2"
+    },
+    "2": {
+      "id": "3",
+      "grant_id": "1",
+      "material_id": "3"
+    },
+    "3": {
+      "id": "4",
+      "grant_id": "1",
+      "material_id": "4"
+    },
+    "4": {
+      "id": "5",
+      "grant_id": "2",
+      "material_id": "5"
+    },
+    "5": {
+      "id": "6",
+      "grant_id": "2",
+      "material_id": "6"
+    },
+    "6": {
+      "id": "7",
+      "grant_id": "2",
+      "material_id": "7"
+    },
+    "7": {
+      "id": "7",
+      "grant_id": "2",
+      "material_id": "8"
+    }
+  },
+  "grants_organizations": {
+    "0": {
+      "id": "1",
+      "grant_id": "1",
+      "organization_id": "1"
+    },
+    "1": {
+      "id": "2",
+      "grant_id": "2",
+      "organization_id": "4"
+    }
+  },
+  "grants_users": {
+    "0": {
+      "id": "1",
+      "grant_id": "1",
+      "user_id": "1"
+    },
+    "1": {
+      "id": "1",
+      "grant_id": "1",
+      "user_id": "2"
+    },
+    "2": {
+      "id": "1",
+      "grant_id": "2",
+      "user_id": "3"
+    }
+  },
+  "guardian_groupobjectpermissions": {
+    "id": "",
+    "object_pk": "",
+    "content_type_id": "",
+    "group_id": "",
+    "permission_id": ""
+  },
+  "guardian_userobjectpermissions": {
+    "id": "",
+    "object_pk": "",
+    "content_type_id": "",
+    "user_id": "",
+    "permission_id": ""
+  },
+  "materials_requests": {
+    "0": {
+      "id": "1",
+      "created_at": "20-01-30 14:12:25-07",
+      "updated_at": "20-01-30 14:12:25-07",
+      "requester_signed_mta_s3_url": "https://bucket-name.s3.region.amazonaws.com/keyname",
+      "irb_s3_url": "https://bucket-name.s3.region.amazonaws.com/keyname",
+      "executed_mta_s3_url": "https://bucket-name.s3.region.amazonaws.com/keyname",
+      "is_active": "false",
+      "status": "closed",
+      "assigned_to_id": "2",
+      "material_id": "2",
+      "requester_id": "3"
+    }
+  },
+  "materials_share_events": {
+    "0": {
+      "id": "1",
+      "created_at": "20-01-30 16:12:25-07",
+      "updated_at": "20-01-30 16:12:25-07",
+      "event_type": "shared",
+      "time": "20-01-30 16:12:25-07",
+      "assigned_to_id": "2",
+      "created_by_id": "1",
+      "material_id": "2"
+    }
+  },
+  "materials": {
+    "0": {
+      "title": "Melanoma Reduction Plasmid",
+      "material_type": "plasmid",
+      "id": "1",
+      "url": "https://www.addgene.org/53246/",
+      "contact_user": "1",
+      "organization_id": "1",
+      "additional_metadata": {
+        "basic_info": {
+          "plasmid_name": "pGP1871",
+          "plasmid_type": "Encodes one insert",
+          "purpose": "Reduces melanoma growth rate",
+          "organism": "Danio rerio",
+          "Number of Availible Samples": "20",
+          "biosafety_level": "2"
+        },
+        "gene_insert": {
+          "gene_insert_name": {
```

Comment:
Note that the interface for this allows users to input multiple values, so multiple values must be supported here.